### PR TITLE
feat(ecc-client-ga4gh-wes): create run component 

### LINKED ISSUE #105: feat: Implement WES component for creating workflow
## Description
[WES](https://github.com/ga4gh/workflow-execution-service-schemas) (Workflow Execution Service) provides an endpoint for creating workflow runs, there is currently no dedicated web component available to facilitate that. This missing component limits the usability and convenience for users who interact with WES through a web interface.

## Possible solution(s)
Add `ecc-client-ga4gh-tes-create-run` to the package created in #94 .

### PR DESCRIPTION
**Pull request by JaeAeich**
## Description
This PR deals with creating a Web Component for POST endpoint of [WES](https://github.com/ga4gh/workflow-execution-service-schemas) for creating a workflow.

Fixes #105 

## Checklist
- [x] My code follows the [contributing guidelines][contributing] of this project.
- [x] I am aware that all my commits will be squashed into a single commit, using the PR title as the commit message.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code in hard-to-understand areas.
- [x] I have updated the user-facing documentation to describe any new or changed behavior.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have not reduced the existing code coverage.

[contributing]: CONTRIBUTING.md

### CODE REVIEW

File: packages/ecc-client-ga4gh-wes/rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      swDest: path.join('dist', 'sw.js'),
+      // directory to match patterns against to be precached
+      globDirectory: path.join('dist'),
+      // cache any html js and css by default
+      globPatterns: ['**/*.{html,js,css,webmanifest}'],
+      skipWaiting: true,
+      clientsClaim: true,
+      runtimeCaching: [{ urlPattern: 'polyfills/*.js', handler: 'CacheFirst' }],
+    }),
+  ],
```
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/rollup.config.js:70]: Why are we generating service workers?

File: packages/ecc-client-ga4gh-wes/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }),
+    /** Bundle assets references via import.meta.url */
+    importMetaAssets(),
+    /** Minify html and css tagged template literals */
+    babel({
+      plugins: [
+        [
+          require.resolve('babel-plugin-template-html-minifier'),
+          {
+            modules: { lit: ['html', { name: 'css', encapsulation: 'style' }] },
```
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/rollup.config.js:42]: lit?

File: packages/ecc-client-ga4gh-wes/rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  },
+  preserveEntrySignatures: false,
+
+  plugins: [
+    /** Enable using HTML as rollup entrypoint */
+    html({
+      minify: true,
+      injectServiceWorker: true,
+      serviceWorkerPath: 'dist/sw.js',
+    }),
```
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/rollup.config.js:26]: Html?

File: packages/ecc-client-ga4gh-wes/rollup.config.js
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/rollup.config.js:1]: If not used you can just use `tsc` for build too!

File: packages/ecc-client-ga4gh-wes/src/components/wes-create-run/definition/wesCreateRun.template.ts
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/src/components/wes-create-run/definition/wesCreateRun.template.ts:1]: Repetitive code!

File: packages/ecc-client-ga4gh-wes/package.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "ecc-client-ga4gh-wes",
+  "description": "Web components for WES",
+  "main": "./out-tsc/src/index.js",
```
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/package.json:4]: Build is right now generating files in `dist`!

File: packages/ecc-client-ga4gh-wes/src/components/wes-run/definition/wesRun.template.ts
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/src/components/wes-run/definition/wesRun.template.ts:1]: Combine the functions here too!

File: packages/ecc-client-ga4gh-wes/src/components/wes-runs/definition/wesRuns.template.ts [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+  'INITIALIZING',
+  'RUNNING',
+  'PAUSED',
+  'COMPLETE',
+  'EXECUTOR_ERROR',
+  'SYSTEM_ERROR',
+  'CANCELED',
+  'CANCELING',
+  'PREEMPTED',
+];
```
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/src/components/wes-runs/definition/wesRuns.template.ts:34]: Keep it at one place!

File: packages/ecc-client-ga4gh-wes/web-test-runner.config.mjs [hunk 6]
```diff
@@ -0,0 +1,41 @@
+// import { playwrightLauncher } from '@web/test-runner-playwright';
+
+const filteredLogs = ['Running in dev mode', 'lit-html is in dev mode'];
```
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-wes/web-test-runner.config.mjs:3]: Remove `lit-html is in dev mode`!

**Review by git-anurag-hub — CHANGES_REQUESTED** [review]
LTGM other that minor fixes!

File: packages/ecc-client-ga4gh-wes/web-test-runner.config.mjs (hunk 6 above)
Comment (JaeAeich) [reply packages/ecc-client-ga4gh-wes/web-test-runner.config.mjs]: done :+1:

File: packages/ecc-client-ga4gh-wes/src/components/wes-runs/definition/wesRuns.template.ts (hunk 5 above)
Comment (JaeAeich) [reply packages/ecc-client-ga4gh-wes/src/components/wes-runs/definition/wesRuns.template.ts]: done :+1:

File: packages/ecc-client-ga4gh-wes/src/components/wes-run/definition/wesRun.template.ts
Comment (JaeAeich) [reply packages/ecc-client-ga4gh-wes/src/components/wes-run/definition/wesRun.template.ts:1]: done :+1:

File: packages/ecc-client-ga4gh-wes/src/components/wes-create-run/definition/wesCreateRun.template.ts
Comment (JaeAeich) [reply packages/ecc-client-ga4gh-wes/src/components/wes-create-run/definition/wesCreateRun.template.ts:1]: remove :+1:

File: packages/ecc-client-ga4gh-wes/src/components/wes-create-run/definition/wesCreateRun.template.ts [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+            name=${(x) => x}
+            @input=${(_, c) => c.parent.handleInput(c.event)}
+            required
+          />
+        </div>
+      `
+    )}
+    <div class="label-input">
+      <label for="workflow_attachment">workflow_attachment</label>
+      <input
```
Comment (SalihuDickson) [inline packages/ecc-client-ga4gh-wes/src/components/wes-create-run/definition/wesCreateRun.template.ts:33]: the api specifies no accepted file types?

Comment (JaeAeich) [reply packages/ecc-client-ga4gh-wes/src/components/wes-create-run/definition/wesCreateRun.template.ts:33]: The parameter type is `formData` and Data type is `Array[string]` :+1:

**Review by git-anurag-hub — APPROVED** [review]

**Pull request merged**